front/lib/api/assistant/configuration.ts

```diff
@@ -983,7 +987,8 @@ export async function createAgentGenerationConfiguration(
     prompt: string; // @todo Daph remove this field
     model: SupportedModel;
     temperature: number;
-  }
+  },
+  agentConfiguration: AgentConfigurationWithoutActionsType
```

Comment:
Question: Did you decide to pass the full `AgentConfigurationWithoutActionsType` instead of just the id to avoid adding a ModelId as param of the function?